<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.55.6" />
  <meta name="author" content="Andrea Rau">

  
  
  
  
  <meta name="description" content="Research Scientist">

  
  <link rel="alternate" hreflang="en-us" href="https://andreamrau.github.io/">

  


  

  
  
  <meta name="theme-color" content="#0095eb">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Arapey:400,400i%7cKarla:400,700%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-133763862-2', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  
  <link rel="alternate" href="https://andreamrau.github.io/index.xml" type="application/rss+xml" title="Andrea Rau, Ph.D.">
  <link rel="feed" href="https://andreamrau.github.io/index.xml" type="application/rss+xml" title="Andrea Rau, Ph.D.">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="https://andreamrau.github.io/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@andreamrau">
  <meta property="twitter:creator" content="@andreamrau">
  
  <meta property="og:site_name" content="Andrea Rau, Ph.D.">
  <meta property="og:url" content="https://andreamrau.github.io/">
  <meta property="og:title" content="Andrea Rau, Ph.D.">
  <meta property="og:description" content="Research Scientist">
  <meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2020-09-30T00:00:00&#43;00:00">
  

  

  <title>Andrea Rau, Ph.D.</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">Andrea Rau, Ph.D.</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#about" data-target="#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#projects" data-target="#projects">
            
            <span>Projects</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#talks" data-target="#talks">
            
            <span>Talks</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#publications" data-target="#publications">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#software" data-target="#software">
            
            <span>Software</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#advising" data-target="#advising">
            
            <span>Advising &amp; Teaching</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#posts" data-target="#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        

        
        
        
        
        

        <li class="nav-item">
          <a href="/#cv" data-target="#cv">
            
            <span>CV</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>


  





  
  
  
  <section id="about" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-telephone person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('https://andreamrau.github.io/img/newportrait_ar.jpg');"></div>
      <meta itemprop="image" content="https://andreamrau.github.io/img/newportrait_ar.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">Andrea Rau</h2>
        <h3 itemprop="jobTitle">Research Scientist</h3>

        
        

        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <a href="https://www.inrae.fr" target="_blank" itemprop="url" rel="noopener">
          <span itemprop="name">INRAE</span>
          </a>
        </h3>
        
      </div>

      <link itemprop="url" href="https://andreamrau.github.io/">

      <ul class="network-icon" aria-hidden="true">
        
        
        <li>
          <a itemprop="sameAs" href="mailto:andrea.rau@inrae.fr" target="_blank" rel="noopener">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//twitter.com/andreamrau" target="_blank" rel="noopener">
            <i class="fa fa-twitter big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="http://scholar.google.com/citations?user=9__bzfgAAAAJ&amp;hl=en" target="_blank" rel="noopener">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="http://www.github.com/andreamrau" target="_blank" rel="noopener">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//orcid.org/0000-0001-6469-488X" target="_blank" rel="noopener">
            <i class="ai ai-orcid big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//researchgate.net/profile/Andrea_Rau" target="_blank" rel="noopener">
            <i class="ai ai-researchgate big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://www.linkedin.com/in/andrea-rau-8133a17/%e2%80%8b" target="_blank" rel="noopener">
            <i class="fa fa-linkedin big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">

    

<h1 id="about-me">About me</h1>

<p>My research interests involve applying statistical reasoning to problems that bring together the fields of biology, genomics, bioinformatics, and statistics. I am especially interested in the development of statistical methodology for the analysis of high-dimensional genomic and transcriptomic data, and the implementation of such methods in open-source R/Bioconductor packages. My research is primarily focused on developing sound statistical methods for the analysis of genomic and transcriptomic data, including differential expression analyses, co-expression analyses, network inference, and integrative multi-omic analyses.</p>

<p>I&rsquo;m currently sharing my time between my home lab in the <a href="https://www6.jouy.inra.fr/gabi_eng/" target="_blank">Animal Genetics and Integrative Biology</a> (GABI) research unit in the <a href="http://www.ga.inra.fr/en" target="_blank">Animal Genetics</a> division of <a href="www.jouy.inra.fr/en" target="_blank">INRAE</a> in Jouy en Josas, France and my host lab in the recently-renamed <a href="http://www.hautsdefrance.inra.fr/Le-centre-Les-recherches/Les-unites-du-centre/AgroImpact" target="_blank">BioEcoAgro</a> research unit (formerly called AgroImpact) in the <a href="http://www.bap.inra.fr/en/" target="_blank">Plant Biology and Breeding Division</a> of <a href="http://www.hautsdefrance.inra.fr/" target="_blank">INRAE</a> in Estrées-Mons, France.</p>


    <div class="row">

      
      <div class="col-sm-5">
        <h3>Interests</h3>
        <ul class="ul-interests">
          
          <li>Biostatistics</li>
          
          <li>Genomics</li>
          
          <li>Transcriptomics</li>
          
          <li><a href="http://www.r-project.org" target="_blank">R</a> and <a href="http://bioconductor.org/" target="_blank">Bioconductor</a></li>
          
        </ul>
      </div>
      

      
      <div class="col-sm-7">
        <h3>Education</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">HDR in Applied Mathematics, 2017</p>
              <p class="institution">Universite d&#39;Evry-Val-d&#39;Essonne</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">PhD In Statistics, 2010</p>
              <p class="institution">Purdue University</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">MS in Applied Statistics, 2007</p>
              <p class="institution">Purdue University</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="projects" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Projects</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    

    

    

    <div class="row isotope projects-container js-layout-masonry">

      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item applications">
        <div class="card">
          
          <a href="https://andreamrau.github.io/project/applications/" title="" class="card-image hover-overlay"
             >
            <img src="/img/applications_illustrate.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://andreamrau.github.io/project/applications/" >Applications</a></h4>
            <div class="card-desription">
              <p>Applications in human, animal, and plant genomics</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item integration">
        <div class="card">
          
          <a href="https://andreamrau.github.io/project/integration/" title="" class="card-image hover-overlay"
             >
            <img src="/img/integrative_illustrate.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://andreamrau.github.io/project/integration/" >Multi-omic data integration</a></h4>
            <div class="card-desription">
              <p>Methodological contributions to integrative multi-omic analyses</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item rnaseq">
        <div class="card">
          
          <a href="https://andreamrau.github.io/project/rnaseq/" title="" class="card-image hover-overlay"
             >
            <img src="/img/coseq_hex.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://andreamrau.github.io/project/rnaseq/" >RNA-seq co-expression</a></h4>
            <div class="card-desription">
              <p>Identify and visualize clusters of co-expressed genes from RNA-seq data</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item networks">
        <div class="card">
          
          <a href="https://andreamrau.github.io/project/networks/" title="" class="card-image hover-overlay"
             >
            <img src="/img/networks_example.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://andreamrau.github.io/project/networks/" >Network inference</a></h4>
            <div class="card-desription">
              <p>Identify gene regulatory networks from transcriptomic data</p>
            </div>
          </div>
        </div>
      </div>
      

    </div>

    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="talks" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent &amp; Upcoming Talks</h1>
    
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2021-04-29_grenoble/">Multi-omic integration for enhanced interpretability in exploratory analyses</a></span>
  <div itemprop="startDate">
    
    Apr 29, 2021
    
      2:00 PM
    
  </div>
  <div class="talk-metadata">
    
            Grenoble Laboratoire Jean Kuntzmann seminar
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://tinyurl.com/Grenoble2021-Rau">
  Slides
</a>




<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2021-04-08_mimo/">Mixture models as a useful tool for identifying co-expressed genes from RNA-seq data</a></span>
  <div itemprop="startDate">
    
    Apr 8, 2021
    
      9:30 AM
    
  </div>
  <div class="talk-metadata">
    
            MiMo Workshop on mixture models
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://tinyurl.com/MiMo2021-Rau">
  Slides
</a>




<a class="btn btn-primary btn-outline btn-xs" href="/project/rnaseq/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2020-07-02_jobim/">Invited keynote: Integrative and interactive analyses of multi-omics data</a></span>
  <div itemprop="startDate">
    
    Jul 2, 2020
    
      2:00 PM
    
  </div>
  <div class="talk-metadata">
    
            JOBIM 2020
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://tinyurl.com/JOBIM2020-Rau">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://relaiswebcasting.mediasite.com/mediasite/Play/a1859271fb2b41d9ab93116f71ae10fb1d?catalog=e534823f0c954836bf85bfa80af2290921">
  Video
</a>



<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2020-05-18_inrae/">Happy 20th Birthday, R!</a></span>
  <div itemprop="startDate">
    
    May 18, 2020
    
      10:00 AM
    
  </div>
  <div class="talk-metadata">
    
            INRAE GiBBS team meeting
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://andreamrau.github.io/2020-05-18_R4_presentation.html">
  Slides
</a>




<a class="btn btn-primary btn-outline btn-xs" href="/project/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2019-12-09_eurobioc/">Invited talk: Individualized multi-omic pathway deviation scores using multiple factor analysis</a></span>
  <div itemprop="startDate">
    
    Dec 9, 2019
    
      10:00 AM
    
  </div>
  <div class="talk-metadata">
    
            EuroBioc 2019
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://speakerdeck.com/andreamrau/individualized-multi-omic-pathway-deviation-scores-using-multiple-factor-analysis">
  Slides
</a>




<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2018-12-04_division/">Exploring drivers of gene expression in The Cancer Genome Atlas</a></span>
  <div itemprop="startDate">
    
    Dec 4, 2018
    
      3:30 PM
    
  </div>
  <div class="talk-metadata">
    
            Division of Biostatistics Seminar at MCW
    
  </div>
  <div class="talk-links">
    







<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2018-06-22_mixstatseq/">Co-expression analyses of RNA-seq data in practice with the R/Bioconductor package coseq</a></span>
  <div itemprop="startDate">
    
    Jun 22, 2018
    
      4:15 PM
    
  </div>
  <div class="talk-metadata">
    
            MixStatSeq Workshop on Mixture Models: Theory and Application
    
  </div>
  <div class="talk-links">
    







<a class="btn btn-primary btn-outline btn-xs" href="/project/rnaseq/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://andreamrau.github.io/talk/2018-03-28_physiology/">Exploring drivers of gene expression in The Cancer Genome Atlas</a></span>
  <div itemprop="startDate">
    
    Mar 28, 2018
    
      8:30 AM
    
  </div>
  <div class="talk-metadata">
    
            Physiology Department Seminar at MCW
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://speakerdeck.com/andreamrau/exploring-drivers-of-gene-expression-in-the-cancer-genome-atlas">
  Slides
</a>




<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="publications" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Publications</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

<h2 id="dissertations-books-book-chapters">Dissertations, books &amp; book chapters</h2>

<ol>
<li><p>Duranthon, V., Araújo, S., Palma, M., <strong>Rau, A.</strong>, Matzapetakis, M., and Almeida, A. (2021) Rabbit research in the post-genomic era: transcriptome, proteome, and metabolome analysis. <em>In: The Genetics and Genomics of the Rabbit.</em>, Ed. L. Fontanesi.</p></li>

<li><p><strong>Rau, A.</strong> (2017) Statistical methods and software for the analysis of transcriptomic data. <em>HDR thesis</em>, Université d’Évry Val-d’Essonne.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/tel-02786130" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/tel-02786130/document" target="_blank" rel="noopener">
PDF
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://speakerdeck.com/andreamrau/hdr-defense-presentation" target="_blank" rel="noopener">
Slides
</a></p></li>

<li><p>Martin-Magniette, M.-L., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2017) Clustering of co-expressed genes. <em>In: Model Choice and Model Aggregation</em>, Ed. F. Bertrand, J.-J. Droesbeke, G. Saporta, C. Thomas-Agnan.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02789928" target="_blank" rel="noopener">
HAL
</a>      <a class="btn btn-primary btn-outline btn-xs"   href="http://www.editionstechnip.com/fr/catalogue-detail/2247/model-choice-and-model-aggregation.html" target="_blank" rel="noopener">
Publisher link
</a></p></li>

<li><p>Albert, I., Ancelet, S., David, O., Denis, J.-B., Makowski, D., Parent, É., <strong>Rau, A.</strong>, and Soubeyrand, S. (2015) Initiation à la statistique bayésienne : Bases théoriques et applications en alimentation, environnement, épidémiologie et génétique. <em>Éditions Ellipses</em>, collection références sciences.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194144" target="_blank" rel="noopener">
HAL
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="http://maiage.jouy.inra.fr/?q=fr/bbb.accueil.html" target="_blank" rel="noopener">
Code
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://www.editions-ellipses.fr/product_info.php?products_id=10293" target="_blank" rel="noopener">
Publisher link
</a></p></li>

<li><p><strong>Rau, A.</strong> (2010) Reverse engineering gene networks using genomic time-course data.. <em>PhD thesis</em>, Purdue University.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/tel-02819388" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://docs.lib.purdue.edu/dissertations/AAI3444841/" target="_blank" rel="noopener">
PDF
</a></p></li>
</ol>

<h2 id="peer-reviewed-publications">Peer-reviewed publications</h2>

<ol>
<li><p>Sellem, E., Marthey, S., <strong>Rau, A.</strong>, Jouneau, L., Bonnet, A., Le Danvic, C., Kiefer, H., Jammes, H., and Schibler, L. (2021) Dynamics of cattle sperm sncRNAs during maturation, from testis to ejaculated sperm. <em>Epigenetics and Chromatin</em>, to appear.</p></li>

<li><p>Mach, N., Moroldo, M., <strong>Rau, A.</strong>, Lecardonnel, J., Le Moyec, L., Robert, C., and Barrey, E. (2021) Understanding the holobiont: crosstalk between gut microbiota and mitochondria during endurance. <em>Frontiers Molecular Biosciences</em>, 8:656204.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03197210" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.frontiersin.org/articles/10.3389/fmolb.2021.656204/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Molecular_Biosciences&id=656204" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/2021.01.08.425889v1" target="_blank" rel="noopener">
Preprint
</a></p></li>

<li><p>Devogel, N., Auer, P. L., Manansala, R., <strong>Rau, A.</strong>, and Wang, T. (2020) A unified linear mixed model for familial relatedness and population structure in genetic association studies. <em>Genetic Epidemiology</em>, 45(3): 305-315.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03176147v1" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/10.1002/gepi.22371" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Cho, Y., <strong>Rau, A.</strong>, Reiner, A., Auer, P. L. (2020) Mendelian randomization analysis with survival outcomes. <em>Genetic Epidemiology</em>, 45(1): 16-23.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03155097v1" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.22354" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Sellem, E., Marthey, S., <strong>Rau, A.</strong>, Jouneau, L., Bonnet, A., Perrier, J.-P., Fritz, S., Le Danvic, C. Boussaha, M., Kiefer, H., Jammes, H., Schiblier, L. (2020) A comprehensive overview of bull sperm-borne small non-coding RNAs and their diversity in six breeds. <em>Epigenetics and Chromatin</em>, 13:19.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03176141v1" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00340-0" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p><strong>Rau, A.</strong>, Manansala, R., Flister, M. J., Rui, H., Jaffrézic, F., Laloë, D., and Auer, P. L. (2020) Individualized multi-omic pathway deviation scores using multiple factor analysis. <em>Biostatistics</em>, kxaa029.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02968360v1" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/biostatistics/advance-article/doi/10.1093/biostatistics/kxaa029/5882080?guestAccessKey=9a15cdfb-100e-4478-8b68-e0a1f9e495f4" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/827022v2" target="_blank" rel="noopener">
Preprint
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/padma/" target="_blank" rel="noopener">
Software
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/RMFRJLA_2019" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Godichon-Baggioni, A., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2020) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. <em>Annals of Applied Statistics</em>, 14:2, 752-767.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789493" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://projecteuclid.org/euclid.aoas/1593449324" target="_blank" rel="noopener">
PDF
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Jehl, F., Désert, C., Klopp, C., Brenet, M., <strong>Rau, A.</strong>, Leroux, S., Boutin, M., Muret, K., Blum, Y., Esquerré, D., Gourichon, D., Burlot, T., Collin, A., Pitel, F., Benani, A., Zerjal, T., Lagarrigue, S. (2019) Chicken adaptive response to low energy diet: main role of the hypothalamic lipid metabolism revealed by a phenotypic and multi-tissue transcriptomic approach. <em>BMC Genomics</em>, 20.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02463367v1" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6384-8" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., <strong>Rau, A.</strong>, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Multi-species annotation of transcriptome and chromatin structure in domesticated animals. <em>BMC Biology</em>, 18:48.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02437599v1" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener">
Preprint
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="http://www.fragencode.org/" target="_blank" rel="noopener">
Website
</a></p></li>

<li><p>Dhara, S., <strong>Rau, A.</strong>, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. <em>Scientific Reports</em>, 9:14198.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02791772" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-019-50485-6" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/638734v3" target="_blank" rel="noopener">
Preprint
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener">
Code
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
Shiny app
</a></p></li>

<li><p><strong>Rau, A.</strong>, Dhara, S., Udvadia, A., and Auer, P. (2019) Regeneration Rosetta: An interactive web application to explore regeneration-associated gene expression and chromatin accessibility. <em>G3: Genes|Genomes|Genetics</em>, 9(12): 3953-3959.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789080" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/12/3953" target="_blank" rel="noopener">
PDF
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener">
Code
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
Shiny app
</a></p></li>

<li><p>Plasterer, C., Tsaih, S.-W., Lemke, A., Schilling, R., Dwinell, M., <strong>Rau, A.</strong>, Auer, P., Rui, H., Flister, M.J. (2019) Identification of a rat mammary tumor risk locus that is syntenic with the commonly amplified 8q12.1 and 8q22.1 regions in human breast cancer patients. <em>G3: Genes|Genomes|Genetics</em>, 9(5): 1739-1743.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02626057" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/5/1739" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Ramayo-Caldas, Y., Zingaretti, L., Bernard, A., Estellé, J. Popova, M., Pons, N., Bellot, P., Mach, N., <strong>Rau, A.</strong>, Roume, H., Perez-Encisco, M., Faverdin, P., Edouard, N., Dusko, S., Morgavi, D.P. and Renand, G. (2019) Identification of rumen microbial biomarkers linked to methane emission in Holstein dairy cows. <em>Journal of Animal Breeding and Genetics</em>, 137:49-59.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02495627" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbg.12427" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p><strong>Rau, A.</strong>, Flister, M. J., Rui, H. and Livermore Auer, P. (2019) Exploring drivers of gene expression in The Cancer Genome Atlas. <em>Bioinformatics</em>, 35(1): 62-68.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02625326" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener">
Preprint
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener">
Code
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener">
Shiny app
</a></p></li>

<li><p>Godichon-Baggioni, A., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. <em>Journal of Applied Statistics</em>, 46(1):47-65.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02789493" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://arxiv.org/abs/1704.06150" target="_blank" rel="noopener">
Preprint
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
Software
</a></p></li>

<li><p><strong>Rau, A.</strong> and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. <em>Briefings in Bioinformatics</em>, bbw128.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02624483" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2016/07/24/065607" target="_blank" rel="noopener">
Preprint
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
Software
</a></p></li>

<li><p>Verrier, E., Genet, C., Laloë, D., Jaffrézic, J., <strong>Rau, A.</strong>, Esquerre, D., Dechamp, N., Ciobataru, C., Hervet, C., Krieg, F., Quillet, E., Boudinot, P. (2018) Genetic and transcriptomic analyses provide new insights on the early antiviral response to VHSV in resistant and susceptible rainbow trout. <em>BMC Genomics</em>, 19:482.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02444518" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4860-1" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Maroilley, T., Berri, M., Lemonnier, G., Esquerré, D., Chevaleyre, C., Mélo, S., Meurens, F., Coville, J.L., Leplat, J.J, <strong>Rau, A.</strong>, Bed’hom, B., Vincent-Naulleau, S., Mercat, M.J., Billon, Y., Lepage, P., Rogel-Gaillard, C., and Estellé, J. (2018) Immunome differences between porcine ileal and jejunal Peyer’s patches revealed by global transcriptome sequencing of gut-associated lymphoid tissues. <em>Scientific Reports</em>, 8:9077.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02356215" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-018-27019-7" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Mondet, F., <strong>Rau, A.</strong>, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018) Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. <em>BMC Genomics</em>, 19:328.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/inserm-02063466" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>He, B., Tjhung, K., Bennett, N., Chou, Y., <strong>Rau, A.</strong>, Huang, J., and Derda, R. (2018) Compositional bias in naïve and chemically-modified phage-displayed libraries uncovered by paired-end deep sequencing. <em>Scientific Reports</em>, 8:1214.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02623801" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://rdcu.be/E4Zi" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Monneret, G., Jaffrézic, F., <strong>Rau, A.</strong>, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. <em>PLoS One</em>, 12(3): e0171142.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01504607" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142" target="_blank" rel="noopener">
PDF
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/Monneret/MarginalCausality" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Sauvage, C., <strong>Rau, A.</strong>, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Glémin, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. <em>The Plant Journal</em>, 91(4):631-645.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01594937" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Rigaill, G., Balzergue, S., Brunaud, V., Blondet, E., <strong>Rau, A.</strong>, Rogier, O., Caius, J., Maugis-Rabusseau, C., Soubigou-Taconnat, L., Aubourg, S., Lurin, C., Martin-Magniette, M.-L., and Delannoy, E. (2016) Synthetic datasets for the identification of key ingredients for RNA-seq differential analysis. <em>Briefings in Bioinformatics</em>, 19(1):65-76.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01595551" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bib/article/doi/10.1093/bib/bbw092/2339782/Synthetic-data-sets-for-the-identification-of-key" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Gallopin, M., Celeux, G., Jaffrézic, F., <strong>Rau, A.</strong> (2015) A model selection criterion for model-based clustering of annotated gene expression data. <em>Statistical Applications in Genetics and Molecular Biology</em>, 14(5): 413-428.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01255908" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml" target="_blank" rel="noopener">
PDF
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/Gallopin/ICAL" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Monneret, G., Jaffrézic, F., <strong>Rau, A.</strong>, Nuel, G. (2015) Estimation d’effets causaux dans les réseaux de régulation génique : vers la grande dimension. <em>Revue d’intelligence artificielle</em>, 29(2): 205-227.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194142" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ria.revuesonline.com/article.jsp?articleId=20894" target="_blank" rel="noopener">
PDF
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p><strong>Rau, A.</strong>, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. <em>Bioinformatics</em>, 31(9): 1420-1427.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01108821" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://hal.archives-ouvertes.fr/hal-01108821" target="_blank" rel="noopener">
Preprint
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/HTSCluster" target="_blank" rel="noopener">
Software
</a></p></li>

<li><p><strong>Rau, A.</strong>, Marot, G. and Jaffrézic, F. (2014) Differential meta-analysis of RNA-seq data from multiple studies. <em>BMC Bioinformatics</em>, 16:31.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00978902" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.biomedcentral.com/1471-2105/15/91/abstract" target="_blank" rel="noopener">
PDF
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank" rel="noopener">
Software
</a></p></li>

<li><p>Endale Ahanda, M.-L., Zerjal, T., Dhorne-Pollet, S., <strong>Rau, A.</strong>, Cooksey, A., and Giuffra, E. (2014) Impact of the genetic background on the composition of the chicken plasma miRNome in response to a stress. <em>PLoS One</em>, 9(12): e114598.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194096" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.plosone.org/article/info/%3Adoi/%2F10.1371/%2Fjournal.pone.0114598" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Nuel, G., <strong>Rau, A.</strong>, and Jaffrézic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments.. <em>Quality Technology and Quantitative Management</em>, 11(1):23-37.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194064" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p><strong>Rau, A.</strong>, Jaffrézic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. <em>BMC Systems Biology</em>, 8:51.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193793" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.biomedcentral.com/1752-0509/7/111/abstract" target="_blank" rel="noopener">
PDF
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Gallopin, M. <strong>Rau, A.</strong>, and Jaffrézic, F. (2013) A hierarchical Poisson log-normal model for network inference from RNA sequencing data. <em>PLoS One</em>, 8(10): e77503.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01004715" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p><strong>Rau, A.</strong>, Gallopin, M., Celeux, G., and Jaffrézic, F. (2013) Data-based filtering for replicated high-throughput transcriptome sequencing experiments. <em>Bioinformatics</em>, 29(17): 2146-2152.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00927025" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bioinformatics.oxfordjournals.org/content/29/17/2146" target="_blank" rel="noopener">
PDF
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank" rel="noopener">
Software
</a></p></li>

<li><p>Dillies, M.-A., <strong>Rau, A.</strong>, Aubert, J., Hennequet-Antier, C., Jeanmougin, M., Servant, N., Keime, C., Marot, G., Castel, D., Estelle, J., Guernec, G., Jagla, B., Jouneau, L., Laloë, D., Le Gall, C., Schaëffer, B., Charif, D., Le Crom, S., Guedj, M., and Jaffrézic, F. (2013) A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. <em>Briefings in Bioinformatics</em>, 14(6): 671-683.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00782486" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bib.oxfordjournals.org/content/14/6/671.abstract?sid=20bf56d5-b2c7-4de2-867d-e533fec9db25" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p>Brenault, P., Lefevre, L. <strong>Rau, A.</strong>, Laloë, D., Pisoni, G., Moroni, P., Bevilacquia, C. and Martin, P. (2013) Contribution of mammary epithelial cells to the immune response during early stages of a bacterial infection to Staphylococcus aureus. <em>Veterinary Research</em>, 45:16.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193839" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.veterinaryresearch.org/content/45/1/16" target="_blank" rel="noopener">
PDF
</a></p></li>

<li><p><strong>Rau, A.</strong>, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2012) Reverse engineering gene regulatory networks using approximate Bayesian computation. <em>Statistics and Computing</em>, 22: 1257-1271.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01019032" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://link.springer.com/article/10.1007/%2Fs11222-011-9309-1" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://arxiv.org/abs/1109.1402" target="_blank" rel="noopener">
Preprint
</a></p></li>

<li><p><strong>Rau, A.</strong>, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2010) An empirical Bayesian method for estimating biological networks from temporal microarray data. <em>Statistical Applications in Genetics and Molecular Biology</em>, 9(1): 9.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193424" target="_blank" rel="noopener">
HAL
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml" target="_blank" rel="noopener">
PDF
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/ebdbNet" target="_blank" rel="noopener">
Software
</a></p></li>

<li><p>Furth, A., Mandrekar, S., Tan, A. <strong>Rau, A.</strong>, Felten, S., Ames, M. Adjei, A. Erlichman, C. and Reid, J. (2008) A limited sample model to predict area under the drug concentration curve for 17-(allylamino)-17-demethoxygeldanamycin and its active metabolite 17-(amino)-17-demethoxygeldanomycin. <em>Cancer Chemotherapy Pharmacology</em>, 61(1): 39-45.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02664062" target="_blank" rel="noopener">
HAL
</a></p></li>
</ol>

<h2 id="pre-prints-technical-reports-and-other-publications">Pre-prints, technical reports, and other publications</h2>

<ol>
<li><strong>Rau, A.</strong> (2021) Cooking up knowledge from big data using data science. <em>Frontiers in Young Minds</em>, 9:632923.
<a class="btn btn-primary btn-outline btn-xs"   href="https://kids.frontiersin.org/article/10.3389/frym.2021.632923" target="_blank" rel="noopener">
PDF
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://dx.doi.org/10.3389/frym.2021.632923" target="_blank" rel="noopener">
Preprint
</a><br /></li>
<li>Mollandin, F., <strong>Rau, A.</strong>, and Croiseau, P. (2021) An evaluation of the interpretability and predictive performance of the BayesR model for genomic prediction., Submitted.
<a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/2020.10.23.351700v1" target="_blank" rel="noopener">
Preprint
</a><br /></li>
<li>Mazo, G., Karlis, D., and <strong>Rau, A.</strong> (2021) A randomized pairwise likelihood method for complex statistical inferences., Submitted.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.archives-ouvertes.fr/hal-03126621" target="_blank" rel="noopener">
Preprint
</a><br /></li>
<li>Cazals, A., Estellé, J., Bruneau, N., Coville, J.-L., Menanteau, P., Rossignol, M.-N., Jardet, D., Bevilacqua, C., <strong>Rau, A.</strong>, Bed’Hom, B., Velge, P., and Calenge, F. (2020) Impact of host genetics on caecal microbiota composition and on Salmonella carriage in chicken., Submitted.
<a class="btn btn-primary btn-outline btn-xs"   href="https://www.researchsquare.com/article/rs-76645/v1" target="_blank" rel="noopener">
Preprint
</a><br /></li>
<li>Bruford, M., Leroy, G., Orozco-terWengel, P., <strong>Rau, A.</strong>, and Simianer H. (2015) Section B: Molecular tools for exploring genetic diversity. <em>The Second Report on the State of the World’s Animal Genetic Resources for Food and Agriculture</em>, FAO Commission on Genetic Resources for Food and Agriculture.
<a class="btn btn-primary btn-outline btn-xs"   href="http://www.fao.org/3/a-i4787e.pdf" target="_blank" rel="noopener">
PDF
</a><br /></li>

<li><p>Nuel, G., <strong>Rau, A.</strong>, and Jaffrézic, F. (2013) Joint likelihood calculation for intervention and observational data from a Gaussian Bayesian network. <em>arXiv</em>, 1305.0709.</p></li>

<li><p><strong>Rau, A.</strong>, Celeux, G., Martin-Magniette, M.-L., and Maugis-Rabusseau, C. (2011) Clustering high-throughput sequencing data with Poisson mixture models. <em>Inria Research Report</em>, 7786.</p></li>

<li><p><strong>Rau, A.</strong>, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2010) Approximate Bayesian approaches for reverse engineering biological networks. <em>Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture</em>, Manhattan, Kansas.</p></li>

<li><p><strong>Rau, A.</strong> (2008) Success of Volunteer Statistical Consulting Service Leads to Expanded Network. <em>The Statistical Consultant</em>, 25(1).</p></li>

<li><p><strong>Rau, A.</strong> (2008) STATCOM Network Engages Growing Number of Student Volunteers. <em>Newsletter for the Section on Statistical Education</em>, 13(1).</p></li>

<li><p><strong>Rau, A.</strong> (2008) Success of Statistical Service Leads to Expanded Network. <em>Amstat News</em>, April 2008.</p></li>
</ol>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="software" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Software</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <ol>
<li><strong>padma</strong>: Pathway deviation scores using multiple factor analysis, available at <a href="https://bioconductor.org/packages/padma/" target="_blank">BioC</a><br /></li>
<li><strong>Invest Astuces</strong>: An R/Shiny interactive web application for financial and real estate loan simulations, available as a <a href="https://investastuces.com/simulateurs/" target="_blank">Shiny web app</a><br /></li>
<li><strong>Regeneration Rosetta</strong>: An R/Shiny interactive web application to explore regeneration-associated gene expression and chromatin accessibility, available as a <a href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank">Shiny web app</a><br /></li>
<li><strong>maskmeans</strong>: Multi-view aggregation/splitting K-means clustering algorithm, available at <a href="https://github.com/andreamrau/maskmeans" target="_blank">GitHub</a><br /></li>
<li><strong>Edge in TCGA</strong>: An R/Shiny interactive web application for the exploration of drivers of gene expression in The Cancer Genome Atlas, available as a <a href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank">Shiny web app</a><br /></li>
<li><strong>coseq</strong>: Co-expression analysis of sequencing data, available at <a href="https://bioconductor.org/packages/coseq/" target="_blank">BioC</a><br /></li>
<li><strong>ICAL</strong>: Model selection for model based clustering of annotated data, available at <a href="https://github.com/Gallopin/ICAL" target="_blank">GitHub</a><br /></li>
<li><strong>metaRNASeq</strong>: Meta-analysis of RNA-seq data, available at <a href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank">CRAN</a><br /></li>
<li><strong>HTSDiff</strong>: Differential analysis for RNA-seq data, available at <a href="https://r-forge.r-project.org/R/?group_id=1504" target="_blank">R-Forge</a><br /></li>
<li><strong>HTSFilter</strong>: Filter for replicated high-throughput sequencing data, available at <a href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank">BioC</a><br /></li>
<li><strong>HTSCluster</strong>: Clustering high-throughput sequencing data with Poisson mixture models, available at <a href="http://cran.r-project.org/web/packages/HTSCluster" target="_blank">CRAN</a><br /></li>
<li><strong>ebdbNet</strong>: Empirical Bayes estimation for dynamic Bayesian networks, available at <a href="http://cran.r-project.org/web/packages/ebdbNet" target="_blank">CRAN</a><br /></li>
</ol>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="advising" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Advising &amp; Teaching</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <ul>
<li><strong>Alexandre Asset</strong> (2021, L2 internship): Inference of co-expression networks from intervention transcriptomic data (with Florence Jaffrézic and Denis Laloë)</li>
<li><strong>Solène Pety</strong> (2021, M1 internship): Knowledge transfer using multivariate gene expression projections onto a large-scale reference database (with Catherine Giauffret)</li>
<li><strong>Fanny Mollandin</strong> (2019-2022, PhD): Incorporating known functional annotations into Bayesian genomic prediction models (with Pascal Croiseau, co-funding from EU Horizon 2020 RIA grant GENE-SWitCH)</li>
<li><strong>Raphaëlle Momal-Leisenring</strong> (2017, M2 internship): Integrative statistical analysis of multi-omics data</li>
<li><strong>Frédéric Jehl</strong> (2017, M2 internship): Impact of heat stress on liver and blood transcriptomes of laying hens (with Tatiana Zerjal)</li>
<li><strong>Dr. Manuel Revilla Sanchez</strong> (2016, 3-month PhD Erasmus+ Learning Mobility): An integrative gene network analysis of the genetic determination of pig fatty acid composition (with Jordi Estelle and Yuliaxis Ramayo Caldas)</li>
<li><strong>Babacar Ciss</strong> (2016, M2 internship): Constructing predictive models for ovine production data (with Eli Sellem, <a href="http://www.allice.fr/" target="_blank">Allice</a>)</li>
<li><strong>Audrey Hulot</strong> (2015, M1 internship): Incorporating a priori biological knowledge into gene network inference from observational and intervention gene expression data (with Florence Jaffrézic)</li>
<li><strong>Meriem Benabbas</strong> (2015, M1 internship): Identifying differentially expressed genes from RNA-seq data using mixture models</li>
<li><strong>Gilles Monneret</strong> (2014-2018, PhD): Estimation of causal effects in gene networks from observational and intervention data (with Grégory Nuel and Florence Jaffrézic)</li>
<li><strong>Marc Teissier and Chaoyu Dong</strong> (2014, M1 STV/EM-ABG internship): Power to detect significantly differential gene expression using RNA-seq data</li>
<li><strong>Mélina Gallopin</strong> (2012-2015, PhD): Clustering and network inference for RNA-seq data (with Gilles Celeux and Florence Jaffrézic)</li>
<li><strong>Rémi Bancal</strong> (2012, M2 internship): Gene network estimation by adaptive knockout experiments (with Grégory Nuel and Florence Jaffrézic)</li>
<li><strong>Mélina Gallopin</strong> (2012, M2 internship): Gene network inference from RNA sequencing expression data (with Gilles Celeux and Florence Jaffrézic)
<br /></li>
</ul>

<hr />

<ul>
<li><strong>Thibault Poinsignon</strong> (2021-2023, PhD advisory committee member)</li>
<li><strong>Nicolas Jouvin</strong> (2021, PhD evaluation committee member (rapportrice))</li>
<li><strong>Lucile Broséus</strong> (2021, PhD evaluation committee member (rapportrice))</li>
<li><strong>Wilfried Heyse</strong> (2020-2022, PhD advisory committee member)</li>
<li><strong>Antoine Leduc</strong> (2019-2022, PhD advisory committee member)</li>
<li><strong>Alyssa Imbert</strong> (2018, PhD evaluation committee member (rapportrice))</li>
<li><strong>Frédéric Jehl</strong> (2017-2020, PhD advisory committee member)</li>
<li><strong>Valentin Voillet</strong> (2016, PhD evaluation committee member)</li>
<li><strong>Gabriel Guillocheau</strong> (2015-2018, PhD advisory committee member)</li>
</ul>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="posts" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">

    <h1>Posts</h1>
    
    
    <p class="view-all">
      <a href="/post/">
        More Posts
        <i class="fa fa-angle-double-right"></i>
      </a>
    </p>
    

  </div>
  <div class="col-xs-12 col-md-8">

    

    
      

      
      

        

        
        

        
        

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2020-09-30 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Sep 30, 2020
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/jobs">jobs</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="https://andreamrau.github.io/post/2020-masters-internships/" itemprop="url">Upcoming Master&#39;s internship offers</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    We are offering two Master’s level internships in early 2021 for candidates witha biostatistics/bioinformatics background.
Please get in touch if you’re interested in either of these opportunities!
A 3-month M1 internship on “Knowledge transfer using multivariate gene expression projectionsonto a large-scale reference database” (location: Inrae Hauts-de-France research centerin Estrées-Mons; full description here).
A 6-month M2 internship on “Improving and extending functionality for multi-omic outlier detection software”(location: Inrae Ile-de-France research center in Jouy-en-Josas orInrae Hauts-de-France research center in Estrées-Mons; full description here).
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="https://andreamrau.github.io/post/2020-masters-internships/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2020-01-03 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Jan 3, 2020
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    4 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/year-in-review">Year in review</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="https://andreamrau.github.io/post/2019-year-in-review/" itemprop="url">2019 year in review</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    Well, here I am again, exactly one year after my last year-in-review post (apparently January 3 is peak procastination time when returning from the holiday break?).
The past year represented a whirlwind of change for me and my family. In the first half of 2019, I wrapped up the final few months of my AgreenSkills+ sabbatical stay as a Visiting Scholar at UWM in Milwaukee, Wisconsin. My time at UWM was exciting, rich in new and continued collaborations, and a wonderful way to expand my research horizons while delving into the world of genomics applied to human health.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="https://andreamrau.github.io/post/2019-year-in-review/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        
          
      

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2019-01-31 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Jan 31, 2019
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="https://andreamrau.github.io/post/tips-and-tricks-for-ggplot2/" itemprop="url">Tips and tricks for ggplot2</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    Like many (most?) users of the ggplot2 visualization package, I often find myself (re-)looking up how to do specific tasks. In an effort to streamline by Googling and avoid searching over and over again for solutions to the same issues, this post will gather together some of the assorted tips and tricks that I’ve recently looked up.
Including an inset graphI found this tip here, using the cowplot package.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="https://andreamrau.github.io/post/tips-and-tricks-for-ggplot2/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2019-01-03 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Jan 3, 2019
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    4 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/year-in-review">Year in review</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="https://andreamrau.github.io/post/2018-year-in-review/" itemprop="url">2018 year in review</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    The start of a new year is always a nice time to look back and take stock of the past year, and look forward and set some goals for the coming year. I spent the entirety of 2018 as an AgreenSkills+ Visiting Scholar at UWM in Milwaukee, Wisconsin, which has been (and continues to be!) a very rich experience that has given me the chance to broaden my understanding of statistical genetics and genomics and expand my skill set.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="https://andreamrau.github.io/post/2018-year-in-review/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-10-31 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Oct 31, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    2 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="https://andreamrau.github.io/post/creating-a-visual-cv/" itemprop="url">Creating a visual CV</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    This is a short post to provide details on how I created the visual CV that is included on my homepage. I got the idea for doing this from a tweet from the awesome Mara Averick about an R package called VisualResume by Nathaniel Phillips:
OMG, I love this! (I miss Breaking Bad so much)
📦 “VisualResume: An R package for creating a visual resume” by @YaRrrBook https://t.co/ZNtbrU87Y4 #rstats pic.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="https://andreamrau.github.io/post/creating-a-visual-cv/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      
    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="cv" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>CV</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <p>Find my full CV in PDF <a href="cv/Rau_Andrea_CV.pdf" target="_blank">here</a>.</p>

<p><img src="cv/Rau_Andrea_visual-CV.png" alt="" /></p>

  </div>
  
</div>

    </div>
  </section>
  



<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2020 &middot; 

      Powered by the
      <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
      <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>


