<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.53" />
  <meta name="author" content="Andrea Rau">

  
  
  
  
  <meta name="description" content="Research Scientist">

  
  <link rel="alternate" hreflang="en-us" href="/">

  


  

  
  
  <meta name="theme-color" content="#328cc1">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-133763862-2', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Andrea Rau, Ph.D.">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="Andrea Rau, Ph.D.">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@andreamrau">
  <meta property="twitter:creator" content="@andreamrau">
  
  <meta property="og:site_name" content="Andrea Rau, Ph.D.">
  <meta property="og:url" content="/">
  <meta property="og:title" content="Andrea Rau, Ph.D.">
  <meta property="og:description" content="Research Scientist">
  <meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2018-08-02T00:00:00&#43;00:00">
  

  

  <title>Andrea Rau, Ph.D.</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">Andrea Rau, Ph.D.</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#about" data-target="#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#projects" data-target="#projects">
            
            <span>Projects</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#talks" data-target="#talks">
            
            <span>Talks</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#publications" data-target="#publications">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#software" data-target="#software">
            
            <span>Software</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#advising" data-target="#advising">
            
            <span>Advising &amp; Teaching</span>
            
          </a>
        </li>

        
        

        
        
        
        
        

        <li class="nav-item">
          <a href="/#cv" data-target="#cv">
            
            <span>CV</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#posts" data-target="#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>


  





  
  
  
  <section id="about" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-telephone person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('/img/portrait_ar.jpg');"></div>
      <meta itemprop="image" content="/img/portrait_ar.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">Andrea Rau</h2>
        <h3 itemprop="jobTitle">Research Scientist</h3>

        
        

        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <a href="http://www.inra.fr" target="_blank" itemprop="url" rel="noopener">
          <span itemprop="name">INRA</span>
          </a>
        </h3>
        
      </div>

      <link itemprop="url" href="/">

      <ul class="network-icon" aria-hidden="true">
        
        
        <li>
          <a itemprop="sameAs" href="mailto:andrea.rau@inra.fr" target="_blank" rel="noopener">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//twitter.com/andreamrau" target="_blank" rel="noopener">
            <i class="fa fa-twitter big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="http://scholar.google.com/citations?user=9__bzfgAAAAJ&amp;hl=en" target="_blank" rel="noopener">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="http://www.github.com/andreamrau" target="_blank" rel="noopener">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//orcid.org/0000-0001-6469-488X" target="_blank" rel="noopener">
            <i class="ai ai-orcid big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//researchgate.net/profile/Andrea_Rau" target="_blank" rel="noopener">
            <i class="ai ai-researchgate big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://www.linkedin.com/in/andrea-rau-8133a17/%e2%80%8b" target="_blank" rel="noopener">
            <i class="fa fa-linkedin big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">

    

<h1 id="biography">Biography</h1>

<p>My research interests involve applying statistical reasoning to problems that bring together the fields of biology, genomics, bioinformatics, and statistics. I am especially interested in the development of statistical methodology for the analysis of high-dimensional genomic and transcriptomic data, and the implementation of such methods in open-source R/Bioconductor packages. My research is primarily focused on developing sound statistical methods for the analysis of genomic and transcriptomic data, including differential expression analyses, co-expression analyses, network inference, and integrative multi-omic analyses.</p>

<p>I&rsquo;m currently on a 20-month sabbatical (October 2017-May 2019) through an <a href="https://www.agreenskills.eu/" target="_blank">AgreenSkills+</a> fellowship to work with my colleague <a href="http://people.uwm.edu/pauer/" target="_blank">Dr. Paul Livermore Auer</a> at the <a href="https://uwm.edu/publichealth/" target="_blank">Zilber School of Public Health</a> at the <a href="https://uwm.edu/" target="_blank">University of Wisconsin-Milwaukee</a>.</p>


    <div class="row">

      
      <div class="col-sm-5">
        <h3>Interests</h3>
        <ul class="ul-interests">
          
          <li>Biostatistics</li>
          
          <li>Genomics</li>
          
          <li>Transcriptomics</li>
          
          <li><a href="http://www.r-project.org" target="_blank">R</a> and <a href="http://bioconductor.org/" target="_blank">Bioconductor</a></li>
          
        </ul>
      </div>
      

      
      <div class="col-sm-7">
        <h3>Education</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">HDR in Applied Mathematics, 2017</p>
              <p class="institution">Universite d&#39;Evry-Val-d&#39;Essonne</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">PhD In Statistics, 2010</p>
              <p class="institution">Purdue University</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">MS in Applied Statistics, 2007</p>
              <p class="institution">Purdue University</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="projects" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Projects</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    
    <span class="hidden default-project-filter">*</span>
    

    
    
    <div class="project-toolbar">
      <div class="project-filters">
        <div class="btn-toolbar">
          <div class="btn-group">
            
              <a href="#" data-filter="*" class="btn btn-primary btn-large active">All</a>
            
              <a href="#" data-filter=".rnaseq" class="btn btn-primary btn-large">RNA-seq co-expression</a>
            
              <a href="#" data-filter=".networks" class="btn btn-primary btn-large">Networks</a>
            
              <a href="#" data-filter=".integration" class="btn btn-primary btn-large">Multi-omic integration</a>
            
          </div>
        </div>
      </div>
    </div>
    
    

    

    <div class="row isotope projects-container js-layout-masonry">

      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item integration">
        <div class="card">
          
          <a href="/project/integration/" title="" class="card-image hover-overlay"
             >
            <img src="/img/edge_in_tcga_figure.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="/project/integration/" >Multi-omic data integration</a></h4>
            <div class="card-desription">
              <p>Integrate multi-omic cancer data representing gene expression, methylation, copy number alterations, somatic mutations, and microRNA expression</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item rnaseq">
        <div class="card">
          
          <a href="/project/rnaseq/" title="" class="card-image hover-overlay"
             >
            <img src="/img/rnaseq-coexpression.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="/project/rnaseq/" >RNA-seq co-expression</a></h4>
            <div class="card-desription">
              <p>Identify clusters of co-expressed genes from RNA-seq data</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item networks">
        <div class="card">
          
          <a href="/project/networks/" title="" class="card-image hover-overlay"
             >
            <img src="/img/networks_example.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="/project/networks/" >Network inference</a></h4>
            <div class="card-desription">
              <p>Identify gene regulatory networks from transcriptomic data</p>
            </div>
          </div>
        </div>
      </div>
      

    </div>

    

  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="talks" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent &amp; Upcoming Talks</h1>
    
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="/talk/2018-12-04_mcw_rau/">Exploring drivers of gene expression in The Cancer Genome Atlas</a></span>
  <div itemprop="startDate">
    
    Dec 4, 2018
    
      3:30 PM
    
  </div>
  <div class="talk-metadata">
    
            Division of Biostatistics Seminar, MCW
    
  </div>
  <div class="talk-links">
    







<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="/talk/2018-03-28_mcw_rau-copy/">Co-expression analyses of RNA-seq data in practice with the R/Bioconductor package coseq</a></span>
  <div itemprop="startDate">
    
    Jun 22, 2018
    
      4:15 PM
    
  </div>
  <div class="talk-metadata">
    
            Mixture models: Theory and applications workshop
    
  </div>
  <div class="talk-links">
    







<a class="btn btn-primary btn-outline btn-xs" href="/project/rnaseq/">
  Project
</a>



  </div>
</div>

      
    
      
        <div class="pub-list-item" style="margin-bottom: 1rem" itemscope itemtype="http://schema.org/Event">
  <i class="fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="/talk/2018-03-28_mcw_rau/">Exploring drivers of gene expression in The Cancer Genome Atlas</a></span>
  <div itemprop="startDate">
    
    Mar 28, 2018
    
      8:30 AM
    
  </div>
  <div class="talk-metadata">
    
            Physiology Department Seminar, MCW
    
  </div>
  <div class="talk-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://speakerdeck.com/andreamrau/exploring-drivers-of-gene-expression-in-the-cancer-genome-atlas">
  Slides
</a>




<a class="btn btn-primary btn-outline btn-xs" href="/project/integration/">
  Project
</a>



  </div>
</div>

      
    
  </div>
</div>

    </div>
  </section>
  

  
  
  
  <section id="publications" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Publications</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

<h2 id="book">Book</h2>

<ul>
<li>Albert, I., Ancelet, S., David, O., Denis, J.-B., Makowski, D., Parent, &Eacute;, <strong>Rau, A.</strong>, and Soubeyrand, S. (2015). <em>Initiation &agrave; la statistique bay&eacute;sienne : Bases th&eacute;oriques et applications en alimentation, environnmenet, &eacute;pid&eacute;miologie et g&eacute;n&eacute;tique</em>. &Eacute;ditions Ellipses, collection r&eacute;f&eacute;rences sciences.
<a class="btn btn-primary btn-outline btn-xs" href="http://maiage.jouy.inra.fr/?q=fr/bbb.accueil.html" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://www.editions-ellipses.fr/product_info.php?products_id=10293" target="_blank" rel="noopener">
Publisher link
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311814" target="_blank" rel="noopener">
ProdInra
</a></li>
</ul>

<h2 id="dissertations">Dissertations</h2>

<ul>
<li><p><strong>Rau, A.</strong> (2017) Statistical methods and software for the analysis of transcriptomic data. HDR (Habilitation &agrave; diriger des recherches) thesis, Universit&eacute; d&rsquo;&Eacute;vry Val-d&rsquo;Essonne.
<a class="btn btn-primary btn-outline btn-xs" href="http://prodinra.inra.fr/ft?id=2922C029-96B2-4A41-9B9B-6FB981ABADA6" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://speakerdeck.com/andreamrau/hdr-defense-presentation" target="_blank" rel="noopener">
Slides
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/408838" target="_blank" rel="noopener">
ProdInra
</a><br />
<em>Note: HDR is a high-level (post-PhD) degree granted by French universities that provides an accreditation to supervise research.</em></p></li>

<li><p><strong>Rau, A.</strong> (2010) Reverse engineering gene networks using genomic time-course data. PhD thesis, Purdue University.
<a class="btn btn-primary btn-outline btn-xs" href="https://docs.lib.purdue.edu/dissertations/AAI3444841/" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/189250" target="_blank" rel="noopener">
ProdInra
</a></p></li>
</ul>

<h2 id="journal-articles">Journal Articles</h2>

<p><center><strong>2019</strong></center></p>

<ul>
<li>Plasterer, C., Tsaih, S.-W., Lemke, A., Schilling, R., Dwinell, M., <strong>Rau, A.</strong>, Auer, P., Rui, H., Flister, M.J. (2019) Identification of a rat mammary tumor risk locus that is syntenic with the commonly amplified 8q12.1 and 8q22.1 regions in human breast cancer patients. <em>G3: Genes|Genomes|Genetics</em> (accepted).</li>
</ul>

<p><center><strong>2018</strong></center></p>

<ul>
<li><p><strong>Rau, A.</strong>, Flister, M. J., Rui, H. and Livermore Auer, P. (2018) Exploring drivers of gene expression in The Cancer Genome Atlas. <em>Bioinformatics</em>, bty551, doi: <a href="https://doi.org/10.1101/227926" target="_blank">https://doi.org/10.1101/227926</a>.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener">
Shiny app
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Godichon-Baggioni, A., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. <em>Journal of Applied Statistics</em>, 46(1):47-65.
<a class="btn btn-primary btn-outline btn-xs" href="https://arxiv.org/abs/1704.06150" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/421247" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p><strong>Rau, A.</strong> and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. <em>Briefings in Bioinformatics</em>, bbw128, <a href="https://doi.org/10.1093/bib/bbw128" target="_blank">https://doi.org/10.1093/bib/bbw128</a>.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2016/07/24/065607" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/413088" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Verrier, E., Genet, C., Lalo&euml;, D., Jaffr&eacute;zic, J.,  <strong>Rau, A.</strong>, Esquerre, D., Dechamp, N., Ciobataru, C., Hervet, C., Krieg, F., Quillet, E., Boudinot, P. (2018) Genetic and transcriptomic analyses provide new insights on the early antiviral response to VHSV in resistant and susceptible rainbow trout. <em>BMC Genomics</em>, 19:482.
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4860-1" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/434908" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Maroilley, T., Berri, M., Lemonnier, G., Esquerr&eacute;, D., Chevaleyre, C., M&eacute;lo, S., Meurens, F., Coville, J.L., Leplat, J.J, <strong>Rau, A.</strong>, Bed&rsquo;hom, B., Vincent-Naulleau, S., Mercat, M.J., Billon, Y., Lepage, P., Rogel-Gaillard, C., and Estell&eacute;, J. (2018). Immunome differences between porcine ileal and jejunal Peyer&rsquo;s patches revealed by global transcriptome sequencing of gut-associated lymphoid tissues. <em>Scientific Reports</em>, 8:9077.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41598-018-27019-7" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/413088" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Mondet, F., <strong>Rau, A.</strong>, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018). Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. <em>BMC Genomics</em>, 19:328.
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/432918" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>He, B., Tjhung, K., Bennett, N., Chou, Y., <strong>Rau, A.</strong>, Huang, J., and Derda, R. (2018). Compositional bias in na&iuml;ve and chemically-modified phage-displayed libraries uncovered by paired-end deep sequencing. <em>Scientific Reports</em>, 8:1214.
<a class="btn btn-primary btn-outline btn-xs" href="http://rdcu.be/E4Zi" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/421248" target="_blank" rel="noopener">
ProdInra
</a></p></li>
</ul>

<p><center><strong>2017</strong></center></p>

<ul>
<li><p>Monneret, G., Jaffr&eacute;zic, F., <strong>Rau, A.</strong>, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. <em>PLoS One</em> 12(3): e0171142.
<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/Monneret/MarginalCausality" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/391068" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Sauvage, C., <strong>Rau, A.</strong>, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Gl&eacute;min, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. <em>The Plant Journal</em> 91(4):631-645.
<a class="btn btn-primary btn-outline btn-xs" href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/393545" target="_blank" rel="noopener">
ProdInra
</a></p></li>
</ul>

<p><center><strong>2016</strong></center></p>

<ul>
<li>Rigaill, G., Balzergue, S., Brunaud, V., Blondet, E., <strong>Rau, A.</strong>, Rogier, O., Caius, J.,  Maugis-Rabusseau, C., Soubigou-Taconnat, L., Aubourg, S., Lurin, C., Martin-Magniette, M.-L., and Delannoy, E. (2016) Synthetic datasets for the identification of key ingredients for RNA-seq differential analysis. <em>Briefings in Bioinformatics</em>, doi: <a href="https://doi.org/10.1093/bib/bbw092" target="_blank">https://doi.org/10.1093/bib/bbw092</a>.
<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bib/article/doi/10.1093/bib/bbw092/2339782/Synthetic-data-sets-for-the-identification-of-key" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/377567" target="_blank" rel="noopener">
ProdInra
</a></li>
</ul>

<p><center><strong>2015</strong></center></p>

<ul>
<li><p>Gallopin, M., Celeux, G., Jaffr&eacute;zic, F., <strong>Rau, A.</strong> (2015) A model selection criterion for model-based clustering of annotated gene expression data. <em>Statistical Applications in Genetics and Molecular Biology</em>, 14(5): 413-428.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/Gallopin/ICAL" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/336851" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Monneret, G., Jaffr&eacute;zic, F., <strong>Rau, A.</strong>, Nuel, G. (2015) Estimation d&rsquo;effets causaux dans les r&eacute;seaux de r&eacute;gulation g&eacute;nique : vers la grande dimension. <em>Revue d&rsquo;intelligence artificielle</em>, 29(2): 205-227.
<a class="btn btn-primary btn-outline btn-xs" href="http://ria.revuesonline.com/article.jsp?articleId=20894" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311809" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p><strong>Rau, A.</strong>, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. <em>Bioinformatics</em>, 31(9): 1420-1427.
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01108821" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/HTSCluster" target="_blank" rel="noopener">
Code
</a>
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311812" target="_blank" rel="noopener">
ProdInra
</a></p></li>
</ul>

<p><center><strong>2014</strong></center></p>

<ul>
<li><p><strong>Rau, A.</strong>, Marot, G. and Jaffr&eacute;zic, F. (2014) Differential meta-analysis of RNA-seq data from multiple studies. <em>BMC Bioinformatics</em>, 15:91.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.biomedcentral.com/1471-2105/15/91/abstract" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/257627" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Endale Ahanda, M.-L., Zerjal, T., Dhorne-Pollet, S., <strong>Rau, A.</strong>, Cooksey, A., and Giuffra, E. (2014) Impact of the genetic background on the composition of the chicken plasma miRNome in response to a stress. <em>PLoS One</em>, 9(12): e114598.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.plosone.org/article/info\%3Adoi\%2F10.1371\%2Fjournal.pone.0114598" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/289922" target="_blank" rel="noopener">
ProdInra
</a></p></li>
</ul>

<p><center><strong>2013</strong></center></p>

<ul>
<li><p>Nuel, G., <strong>Rau, A.</strong>, and Jaffr&eacute;zic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments. <em>Quality Technology and Quantitative Management</em> 11(1):23-37.
<a class="btn btn-primary btn-outline btn-xs" href="http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/283565" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p><strong>Rau, A.</strong>, Jaffr&eacute;zic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. <em>BMC Systems Biology</em> 7:111.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.biomedcentral.com/1752-0509/7/111/abstract" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/255885" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Gallopin, M. <strong>Rau, A.</strong>, and Jaffr&eacute;zic, F. (2013). A hierarchical Poisson log-normal model for network inference from RNA sequencing data. <em>PLoS One</em> 8(10): e77503.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/221846" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p><strong>Rau, A.</strong>, Gallopin, M., Celeux, G., and Jaffr&eacute;zic, F. (2013). Data-based filtering for replicated high-throughput transcriptome sequencing experiments. <em>Bioinformatics</em> 29(17): 2146-2152.
<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/29/17/2146" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/201701" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Dillies, M.-A.&sup1;, <strong>Rau, A.&sup1;</strong>, Aubert, J.&sup1;, Hennequet-Antier, C.&sup1;, Jeanmougin, M.&sup1;, Servant, N.&sup1;, Keime, C.&sup1;, Marot, G., Castel, D., Estelle, J., Guernec, G., Jagla, B., Jouneau, L., Lalo&euml;, D., Le Gall, C., Scha&euml;ffer, B., Charif, D., Le Crom, S.&sup1;, Guedj, M.&sup1;, and Jaffr&eacute;zic, F&sup1;. (2013). A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. <em>Briefings in Bioinformatics</em>, doi:10.1093/bib/bbs046.
<a class="btn btn-primary btn-outline btn-xs" href="http://bib.oxfordjournals.org/content/14/6/671.abstract?sid=20bf56d5-b2c7-4de2-867d-e533fec9db25" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/173448" target="_blank" rel="noopener">
ProdInra
</a><br />
&sup1;These authors contributed equally to this work.</p></li>

<li><p>Brenault, P., Lefevre, L. <strong>Rau, A.</strong>, Lalo&euml;, D., Pisoni, G., Moroni, P., Bevilacquia, C. and Martin, P. (2013) Contribution of mammary epithelial cells to the immune response during early stages of a bacterial infection to Staphylococcus aureus. <em>Veterinary Research</em>, 45:16.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.veterinaryresearch.org/content/45/1/16" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/263994" target="_blank" rel="noopener">
ProdInra
</a></p></li>
</ul>

<p><center><strong>2012 and before</strong></center></p>

<ul>
<li><p><strong>Rau, A.</strong>, Jaffr&eacute;zic, F., Foulley, J.-L., and Doerge, R. W. (2012). Reverse engineering gene regulatory networks using approximate Bayesian computation. <em>Statistics and Computing</em>, 22: 1257-1271.
<a class="btn btn-primary btn-outline btn-xs" href="https://arxiv.org/abs/1109.1402" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://link.springer.com/article/10.1007\%2Fs11222-011-9309-1" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/172363" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p><strong>Rau, A.</strong>, Jaffr&eacute;zic, F., Foulley, J.-L., and Doerge, R. W. (2010). An empirical Bayesian method for estimating biological networks from temporal microarray data. <em>Statistical Applications in Genetics and Molecular Biology</em>: Vol. 9: Iss. 1, Article 9.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml" target="_blank" rel="noopener">
PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/ebdbNet" target="_blank" rel="noopener">
Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/6927" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Furth, A., Mandrekar, S., Tan, A. <strong>Rau, A.</strong>, Felten, S., Ames, M. Adjei, A. Erlichman, C. and Reid, J. (2008). A limited sample model to predict area under the drug concentration curve for 17-(allylamino)-17-demethoxygeldanamycin and its active metabolite 17-(amino)-17-demethoxygeldanomycin. <em>Cancer Chemotherapy Pharmacology</em>, 61(1): 39-45.</p></li>
</ul>

<h2 id="book-chapters">Book chapters</h2>

<ul>
<li>Martin-Magniette, M.-L., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2017) Clustering of co-expressed genes. In: <em>Model Choice and Model Aggregation</em>. Ed. F. Bertrand, J.-J. Droesbeke, G. Saporta, C. Thomas-Agnan.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.editionstechnip.com/fr/catalogue-detail/2247/model-choice-and-model-aggregation.html" target="_blank" rel="noopener">
Publisher link
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01943296" target="_blank" rel="noopener">
HAL
</a></li>
</ul>

<h2 id="submitted-and-in-preparation">Submitted and in preparation</h2>

<ul>
<li><p>Dhara, S., <strong>Rau, A.</strong>, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Regulation of CNS regeneration-assocoiated genes is driven by a temporally changing cast of transcription factors. <em>bioRxiv</em>.<br />
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
Shiny app
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p><strong>Rau, A.</strong>, Dhara, S., Udvadia, A., and Auer, P. (2019) <em>Regeneration Rosetta</em>: An interactive web application to explore
regeneration-associated gene expression and chromatin accessibility. <em>bioRxiv</em>.<br />
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
Shiny app
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Revilla, M., <strong>Rau, A.</strong>, Crespo-Piazuelo, D., Ramayo-Caldas, Y., Estell&eacute;, J., INIA, Ballester, M., Folch, J. M. (2019) An integrative gene network analysis of the genetic determination of pig fatty acid composition based on adipose tissue RNA sequencing. <em>Submitted.</em></p></li>

<li><p>Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., <strong>Rau, A.</strong>, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2018) Livestock genome annotation: transcriptome and chromatin structure profiling in cattle, goat, chicken, and pig. <em>bioRxiv</em>, doi: <a href="https://doi.org/10.1101/316091" target="_blank">https://doi.org/10.1101/316091</a>. <em>Submitted.</em>
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/441070" target="_blank" rel="noopener">
ProdInra
</a></p></li>

<li><p>Godichon-Baggioni, A., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2018) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. <em>Submitted.</em>
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01916941" target="_blank" rel="noopener">
Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener">
Code
</a></p></li>

<li><p>Tsaih, S.-W., Plasterer, C., Lemke, A., Ran, S. <strong>Rau, A.</strong>, Auer, P., Rui, H. and Flister, M. J. (2018) Genetic mapping of pathophysiological modifiers in the breast tumor microenvironment. <em>Submitted</em>.</p></li>

<li><p>Ramayo-Caldas, Y., Zingaretti, L., Bernard, A., Estell&eacute;, J. Popova, M., Pons, N., Bellot, P., Mach, N., <strong>Rau, A.</strong>, Roume, H., Perez-Encisco, M., Faverdin, P., Edouard, N., Dusko, S., Morgavi, D.P. and Renand, G. (2019) Identification of rumen microbial biomarkers linked to methane emission in Holstein dairy cows. <em>Submitted.</em></p></li>

<li><p>Jehl, F., Klopp, C., Brenet, M., <strong>Rau, A.</strong>, D&eacute;sert, C., Boutin, M., Leroux, S., Muret, K.,  Esquerr&eacute;, D., Gourichon, D., Burlot, T., Pitel, F., Zerjal, T., Lagarrigue, S. (2018) Phenotype and multi-tissue transcriptome response to diet energy change in laying hens. <em>In preparation.</em></p></li>
</ul>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="software" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Software</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <ul>
<li><a href="https://investastuces.com/simulateurs/" target="_blank"><strong>Invest Astuces</strong></a>: An R/Shiny interactive web application for financial and real estate loan simulations</li>
<li><a href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank"><strong>Regeneration Rosetta</strong></a>: An R/Shiny interactive web application to explore
regeneration-associated gene expression and chromatin accessibility</li>
<li><a href="https://github.com/andreamrau/maskmeans" target="_blank"><strong>maskmeans</strong></a>: Multi-view aggregation/splitting K-means clustering algorithm.</li>
<li><a href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank"><strong>Edge in TCGA</strong></a>: An R/Shiny interactive web application for the exploration of drivers of gene expression in <a href="https://cancergenome.nih.gov/" target="_blank">The Cancer Genome Atlas</a>.</li>
<li><a href="https://bioconductor.org/packages/coseq/" target="_blank"><strong>coseq</strong></a>: Co-expression analysis of sequencing data.</li>
<li><a href="https://github.com/Gallopin/ICAL" target="_blank"><strong>ICAL</strong></a>: Model selection for model based clustering of annotated data.</li>
<li><a href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank"><strong>metaRNASeq</strong></a>: Meta-analysis of RNA-seq data.</li>
<li><a href="https://r-forge.r-project.org/R/?group_id=1504" target="_blank"><strong>HTSDiff</strong></a>: Differential analysis for RNA-seq data.</li>
<li><a href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank"><strong>HTSFilter</strong></a>: Filter for replicated high-throughput sequencing data.</li>
<li><a href="http://cran.r-project.org/web/packages/HTSCluster" target="_blank"><strong>HTSCluster</strong></a>: Clustering high-throughput sequencing data with Poisson mixture models.</li>
<li><a href="http://cran.r-project.org/web/packages/ebdbNet" target="_blank"><strong>ebdbNet</strong></a>: Empirical Bayes estimation for dynamic Bayesian networks.</li>
</ul>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="advising" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Advising &amp; Teaching</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <p>I am an adjunct instructor for the following graduate course at the Medical College of Wisconsin in Spring 2019:</p>

<ul>
<li><a href="https://www.mcw.edu/departments/physiology/education" target="_blank">MCW</a> Physiological Genomics: Bioinformatics module</li>
</ul>

<p>I was a teaching instructor for the following course at the University of Wisconsin-Milwaukee in Spring 2018:</p>

<ul>
<li><a href="https://uwm.edu/" target="_blank">UWM</a> PH718: Data management and visualization</li>
</ul>

<hr />

<p><strong>Alumni</strong>:</p>

<ul>
<li><strong>Dr. Gilles Monneret</strong> (2014-2018 Ph.D.): &ldquo;Estimation of causal effects in gene networks from observational and intervention data&rdquo; (co-supervision with <a href="http://nuel.perso.math.cnrs.fr/" target="_blank">Gr&eacute;gory Nuel</a> and <a href="http://www6.jouy.inra.fr/gabi_eng/Who-are-we/Directory-of-GABI-members/JAFFREZIC-florence" target="_blank">Florence Jaffr&eacute;zic</a>)</li>
<li><strong>Rapha&euml;lle Momal-Leisenring</strong> (2017 M2 internship): &ldquo;Integrative statistical analysis of multi-omics data&rdquo;</li>
<li><strong>Fr&eacute;d&eacute;ric Jehl</strong> (2017 M2 internship): &ldquo;Impact of heat stress on liver and blood transcriptomes of laying hens&rdquo; (co-supervision with Tatiana Zerjal)</li>
<li><strong>Dr. Manuel Revilla Sanchez</strong> (2016 3-month Ph.D. Erasmus+ Learning Mobility): &ldquo;An integrative gene network analysis of the genetic determination of pig fatty acid composition&rdquo; (co-supervison with Jordi Estelle and Yuliaxis Ramayo Caldas)</li>
<li><strong>Babacar Ciss</strong> (2016 M2 internship): &ldquo;Constructing predictive models for ovine production data&rdquo; (co-supervision with Eli Sellem, <a href="http://www.allice.fr/" target="_blank">Allice</a>)</li>
<li><a href="http://www.math.u-psud.fr/~gallopin/" target="_blank"><strong>Dr. M&eacute;lina Gallopin</strong></a> (2012-2015 Ph.D.): &ldquo;Clustering and network inference for RNA-seq data&rdquo; (co-supervision with <a href="http://www.math.u-psud.fr/select/people/celeux/Welcome.html" target="_blank">Gilles Celeux</a> and <a href="http://www6.jouy.inra.fr/gabi_eng/Who-are-we/Directory-of-GABI-members/JAFFREZIC-florence" target="_blank">Florence Jaffr&eacute;zic</a>)
Currently Assistant Professor (ma&icirc;tre de conf&eacute;rences) at <a href="http://www.i2bc.paris-saclay.fr/" target="_blank">I2BC, Universit&eacute; Paris-Saclay</a></li>
<li><strong>Audrey Hulot</strong> (2015 M1 internship): &ldquo;Incorporating a priori biological knowledge into gene network inference from observational and intervention gene expression data&rdquo; (with <a href="http://www6.jouy.inra.fr/gabi_eng/Who-are-we/Directory-of-GABI-members/JAFFREZIC-florence" target="_blank">Florence Jaffr&eacute;zic</a>)</li>
<li><strong>Meriem Benabbas</strong> (2015 M1 internship): &ldquo;Identifying differentially expressed genes from RNA-seq data using mixture models&rdquo;</li>
<li><strong>R&eacute;mi Bancal</strong> (2012 M2 internship): &ldquo;Gene network estimation by adaptive knockout experiments&rdquo; (co-supervision with <a href="http://nuel.perso.math.cnrs.fr/" target="_blank">Gr&eacute;gory Nuel</a> and <a href="http://www6.jouy.inra.fr/gabi_eng/Who-are-we/Directory-of-GABI-members/JAFFREZIC-florence" target="_blank">Florence Jaffr&eacute;zic</a>)</li>
<li><strong>M&eacute;lina Gallopin</strong> (2012 M2 internship): &ldquo;Gene network inference from RNA sequencing expression data&rdquo; (co-supervision with <a href="http://www.math.u-psud.fr/select/people/celeux/Welcome.html" target="_blank">Gilles Celeux</a> and <a href="http://www6.jouy.inra.fr/gabi_eng/Who-are-we/Directory-of-GABI-members/JAFFREZIC-florence" target="_blank">Florence Jaffr&eacute;zic</a>)</li>
</ul>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="cv" class="home-section">
    <div class="container">
      


<div class="row">
  
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>CV</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <p>Find my full CV in PDF <a href="cv/Rau_Andrea_CV.pdf" target="_blank">here</a>.</p>

<p><img src="cv/Rau_Andrea_visual-CV.png" alt="" /></p>

  </div>
  
</div>

    </div>
  </section>
  

  
  
  
  <section id="posts" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">

    <h1>Posts</h1>
    
    

  </div>
  <div class="col-xs-12 col-md-8">

    

    
      

      
      

        

        
        

        
        
          
      

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2019-01-31 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Jan 31, 2019
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="/post/tips-and-tricks-for-ggplot2/" itemprop="url">Tips and tricks for ggplot2</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    Like many (most?) users of the ggplot2 visualization package, I often find myself (re-)looking up how to do specific tasks. In an effort to streamline by Googling and avoid searching over and over again for solutions to the same issues, this post will gather together some of the assorted tips and tricks that I’ve recently looked up.
Including an inset graph I found this tip here, using the cowplot package.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="/post/tips-and-tricks-for-ggplot2/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2019-01-03 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Jan 3, 2019
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    4 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/year-in-review">Year in review</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="/post/2018-year-in-review/" itemprop="url">2018 year in review</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    The start of a new year is always a nice time to look back and take stock of the past year, and look forward and set some goals for the coming year. I spent the entirety of 2018 as an AgreenSkills+ Visiting Scholar at UWM in Milwaukee, Wisconsin, which has been (and continues to be!) a very rich experience that has given me the chance to broaden my understanding of statistical genetics and genomics and expand my skill set.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="/post/2018-year-in-review/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-10-31 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Oct 31, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    2 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="/post/creating-a-visual-cv/" itemprop="url">Creating a visual CV</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    This is a short post to provide details on how I created the visual CV that is included on my homepage. I got the idea for doing this from a tweet from the awesome Mara Averick about an R package called VisualResume by Nathaniel Phillips:
OMG, I love this! (I miss Breaking Bad so much)
📦 “VisualResume: An R package for creating a visual resume” by @YaRrrBook https://t.co/ZNtbrU87Y4 #rstats pic.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="/post/creating-a-visual-cv/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        
          
      

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-08-03 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Aug 3, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    3 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="/post/alpha-transparency-in-ggplot2/" itemprop="url">Mapping transparency in ggplot2 to a continuous variable</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    tl;dr: Use I() to treat a numeric variable in a data.frame “as is” and avoid unintended conversion when mapping to transparency in a ggplot2 aesthetic. Today I ran into a ggplot2 plotting problem involving mapping the transparency aesthetic to a numeric variable – this drove me crazy until I figured it out.
Here’s the basic set-up: I wanted to plot a scatterplot of two variables, but have the transparency of the points be controlled by a third (numeric) variable.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="/post/alpha-transparency-in-ggplot2/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      

        

        
        

        
        
          
      

      
      
        
        

<div class="card-simple" itemscope itemprop="blogPost" itemtype="http://schema.org/BlogPosting">
  

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-02-22 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Feb 22, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Andrea Rau">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    4 min read
  </span>
  

  
  

  
  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  

</div>


  
  
  <h3 class="article-title" itemprop="headline">
    <a href="/post/hello-blogdown/" itemprop="url">Setting up blogdown</a>
  </h3>
  <div class="article-style" itemprop="articleBody">
    
    I recently decided that I wanted to move my professional homepage from a free page set up on WordPress to GitHub Pages using blogdown by Yihui Xi.
There were basically two reasons for this: (1) Because I only sprang for the free WordPress site, there are gigantic, ugly ads that appear on every single page. I only recently realized this as I was usually viewing my WordPress site while being logged on – and apparently, the ads only appear for other people.
    
  </div>
  <p class="read-more" itemprop="mainEntityOfPage">
    <a href="/post/hello-blogdown/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      

      
    

  </div>
</div>

    </div>
  </section>
  



<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Powered by the
      <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
      <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>


